src/backend/distributed/template.awk

```diff
@@ -0,0 +1,24 @@
+# https://stackoverflow.com/a/51808896/2570866
```

Comment:
An alternative that avoids our own build script is to use the C preprocessor & `#include`. This also opens up being able to reference `PG_VERSION_NUM`

See https://github.com/citusdata/postgresql-hll/pull/73/commits/8d4b4c7fda9a3c62cbbc42407707d999f880705d where I implemented this for postgresql-hll

Reply:
That's a good idea. Definitely open to do the same for this PR.

Reply:
Done